src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -212,6 +358,18 @@ def wrapper(*args, **kwargs):
                     #  Handle retriable exception, please refer to
                     #  https://platform.openai.com/docs/guides/error-codes/api-errors
                     print(f"Exception occurs: {type(e).__name__}: {str(e)}", file=sys.stderr)
+                    # Handle llm use some unsupported model like vision error
```

Comment:
Vision model does not support all chat api parameters, e.g. `response_format` and `function_call`.
Recommend user to use vision model in vision tools, rather than LLM tool.